### PR TITLE
build images with --norun

### PR DESCRIPTION
**Pull request by Caribosaurus**
Currently rules_docker have different outcomes when doing a bazel run. When running a Container_image it will only tag the image but running a language specific image (I.E go_image) it will also do a docker run. This behavior forces users of such rules to run with -- --norun in order to simply tag the image (The tilt expected outcome).

Running with --norun allows tilt to detect that build has ended.

### CODE REVIEW

File: Tiltfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-BAZEL_RUN_CMD = "bazel run --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 %s"
```
Comment (jazzdan) [inline Tiltfile:1]: Why did you remove this platforms flag?

**Review by jazzdan — COMMENTED** [review]
Thanks for helping us keep these up to date! One question:

File: Tiltfile (hunk 1 above)
Comment (Caribosaurus) [reply Tiltfile:1]: I simply don't think it is needed. Perhaps I'm wrong. Is this plateform the only one supported?

Comment (jazzdan) [reply Tiltfile:1]: It's been a while since I looked but my memory is that without specifying the platform I would end up with say macOS binaries in a Linux container if I was running Tilt on macOS. Then the Linux container wouldn't be able to start them.

Of course now I don't have a macOS machine to test with so I can't be sure. @nicks can you confirm?

Comment (Caribosaurus) [reply Tiltfile:1]: @jazzdan That makes total sense. Maybe it should be mentionned in the [bazel documentation page](https://docs.tilt.dev/integrating_bazel_with_tilt.html)

Comment (jazzdan) [reply Tiltfile:1]: Good idea! I created an issue for that [here](https://github.com/tilt-dev/tilt.build/issues/626).

Comment (nicks) [reply Tiltfile:1]: @jazzdan ya, gmas and I'll test on my macos box

Comment (nicks) [reply Tiltfile:1]: yep, confirmed! the `--platforms` line is necessary. if you remove it, you get exec format errors because it tries to copy a macos binary to the linux container

**Review by jazzdan — APPROVED** [review]
LGTM!

**Comment by nicks** [comment]
i don't know why this never got merged! merging now
